tests/particle_output_multiple_formats/screen-output

```diff
@@ -16,40 +11,20 @@ Number of degrees of freedom: 4,645 (2,178+289+1,089+1,089)
    Solving Stokes system... 30+3 iterations.
 
    Postprocessing:
-     Writing particle output: output/particles/particles-00001.{txt,vtu,hdf5}
+     Writing particle output: output-particle_output_multiple_formats/particles/particles-00000.{txt,h5,vtu}
```

Comment:
This is the output format that I have currently implemented, `particles-00000.{txt,h5,vtu}`.
If only one format is specified, for example `ascii`,  the string is set to `particles-00000.txt`.
Is this clear cut?